README.md

```diff
@@ -1,11 +1,53 @@
 # ingress53
 
-ingress53 is a service designed to run in kubernetes and maintain DNS records for its ingress resources on AWS Route53.
+ingress53 is a service designed to run in kubernetes and maintain DNS records for the cluster's ingress resources in AWS Route53.
+
+It will watch the kubernetes API (using the service token) for any Ingress resource changes and try to apply those records to route53 in Amazon mapping the record to the "target name", which is the dns name of the ingress endpoint for your cluster.
```

Comment:
I don't think so, this does not need ingress controller to work, people can use something like `nodePort` and have this work for them.